Transcripts/EP_27_Emma_Bostian.md

```diff
@@ -1,716 +1,210 @@
+
+**Emma:**: [00:41:29]:
+I wish that I had better advice in terms of like what you should do, but my advice isn't necessarily going to work for everyone. So, you know, I would just say like, look, look for people that you can trust and try to be open with them and get advice from them if they've been through something similar, but just know that it's not okay to feel like put down at work or unsafe at work. I think too, if you are in a place of privilege, like you need to use that for good. I've seen nasty comments in Slack, not at this job at previous jobs. And it did not sit well with me. And I thought, well, I have two options. I can say something and hopefully change the future, or I can sit here and be compliant. And if you are silent, you're compliant or you're complicit. And I chose to say something and guess what? Yeah. People, some people might not like you, they might think that you're lame that you're, you know, not fun. It's just a joke. But in the workplace, if a joke is demeaning to someone, it's not a joke. And so I've been that person to like, contact my manager and say, Hey, this didn't sit well with me. Like, what are we going to do about it? So if you have that privilege to be able to do that, you need to be doing it.
+
+**Michaela:**: [00:42:41]:
+Yeah, I totally and wholeheartedly agree. And I also think this is really, really good advice and even a wonderful ending for today's episode. So, Emma, is there something else you want to say to my listeners to wrap up this episode?
+
+**Emma:**: [00:42:56]:
+So I kind of just want to say, like, if you're listening and you're struggling, I'm right there with you, we're all struggling in different ways. So we just gotta be there for each other.
+
+**Michaela:**: [00:43:05]:
+Yeah. That's really nice. That's so true. And I will link to all of your core stuff, your Twitter profile, of course, to your book, we talked a little bit about that, right. And so while then I say, thank you, Emma, for, for being on my show. Thank you so much for being here. Yeah. Thank you for having me. Yeah, it was really wonderful. I really enjoyed it. I hope you enjoyed another episode of the sup engineering unlocked podcast. Don't forget to subscribe. And I talk to you again in two weeks. Bye.
```

Comment:
This part for example was already improved in a previous version. 

```suggestion
Yeah. That's really nice. That's so true. And I will link to all of your cool stuff: your Twitter profile, of course, to your book, we talked a little bit about that, right. And so while then I say, thank you, Emma, for, for being on my show. Thank you so much for being here. 

**Emma**: Yeah. Thank you for having me. 

**Michaela**: Yeah, it was really wonderful. I really enjoyed it. I hope you enjoyed another episode of the software engineering unlocked podcast. Don't forget to subscribe, and I talk to you again in two weeks. Bye.
```

Reply:
Hi @mgreiler, thanks for getting back to me. Yeah, I've been having issues with my account being marked as spam by github scripts, which I'm guessing has hidden all my activity, hence why things have fallen out of sync :( 

I did use VS Code to limit each line to 80 characters, so I dunno why it's not followed my PR... 

I'll be in touch :)

Reply:
Yeah, I got an email about your activity, but when I wanted to see it (i.e., click the link and engage on Github) I got a 404 and wondered what is going on. 

Falling out of sync can be quite a challenge. I hope you manage to merge your changes with the latest version.
The 80 chars limit can help with it a bit.

Reply:
BTW, if it's messy to integrate your changes, it might actually be easier to pick another episode. Just a thought... no pressure.

Reply:
Yeah, I completely agree. I think I like the look of Ep 20 - #32 - could you please assign it to me - I'll put a message there too... Thanks.